recipe/emailverification/utils.go

```diff
@@ -6,25 +6,34 @@ import (
 )
 
 func ValidateAndNormaliseUserInput(appInfo supertokens.NormalisedAppinfo, config schema.TypeInput) schema.TypeNormalisedInput {
-	getEmailVerificationURL := *config.GetEmailVerificationURL
+	var typeNormalisedInput schema.TypeNormalisedInput
+	typeNormalisedInput.GetEmailVerificationURL = *config.GetEmailVerificationURL
 	if config.GetEmailVerificationURL == nil {
-		getEmailVerificationURL = GetEmailVerificationURL(appInfo)
+		typeNormalisedInput.GetEmailVerificationURL = GetEmailVerificationURL(appInfo)
 	}
-	createAndSendCustomEmail := *config.CreateAndSendCustomEmail
+	typeNormalisedInput.CreateAndSendCustomEmail = *config.CreateAndSendCustomEmail
 	if config.CreateAndSendCustomEmail == nil {
-		createAndSendCustomEmail = CreateAndSendCustomEmail(appInfo)
+		typeNormalisedInput.CreateAndSendCustomEmail = CreateAndSendCustomEmail(appInfo)
```

Comment:
The functions that we implement like `CreateAndSendCustomEmail`, perhaps call them `DefaultCreateAndSendCustomEmail` or something like that.. so it's clear as to what's going on..